boards/arm/mps2_an385/mps2_an385_defconfig

```diff
@@ -28,3 +28,10 @@ CONFIG_UART_CMSDK_APB=y
 CONFIG_WATCHDOG=y
 
 CONFIG_I2C=y
+
+#userspace
+CONFIG_ARCH_HAS_USERSPACE=y
+CONFIG_ARM_MPU=y
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
Why do we need to override the main/idle/privileged stack size from defaults?

Reply:
Qemu cant handle stacks lesser than 1024.  This will be a region i need to congfigure in the MPU and the qemu has a 1k limitation.

Reply:
then why aren't you changing this in qemu_cortex_m3 board definition?

Reply:
These stack size changes need to be moved out of the mps2_an385 configuration, I can't merge this since this is really something that affects QEMU and not the actual mps2_an385 board.

Reply:
@AdithyaBaglody you never addressed the comment above.

Reply:
oh I had actually addressed it before. Now after enabling the userspace(in the last 2days) this came back in.
will fix it. Thanks for catching this.